docs/rest-api/rest-interpreter.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
-title: "Interpreter REST API"
-description: ""
+title: "Apache Zeppelin Interpreter REST API"
+description: "Apache Zeppelin provides several REST APIs for interaction and remote activation of zeppelin functionality. This page contains Apache Zeppelin Interpreter REST API information."
```

Comment:
What is `remote activation of zeppelin` ?

May be `This page contains Apache Zeppelin Interpreter REST API information.` could be enough here